src/hooks/mongodb3x.test.js

```diff
@@ -1,31 +1,33 @@
 import { HandlerInputsBuilder } from '../../testUtils/handlerInputsBuilder';
```

Comment:
Generally comment, this file should not be changed. Changing the tests make us work hard to check if the old logic is broken

Reply:
it's a good point, but there are no behaviour changes here, just cleanup. if you'd rather i returned it let me know